src/libs/fileDownload/index.ts

```diff
@@ -29,6 +29,7 @@ const fileDownload: FileDownload = (url, fileName) => {
             // adding href to anchor
             link.href = href;
             link.style.display = 'none';
+            // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing -- nullish coalescing doesn't achieve the same result in this case
```

Comment:
could you expand on this a little bit more?

Reply:
Sure, in a new commit I expanded the comment a bit, but I can elaborate more. This function is called in multiple places in the code and sometimes fileName can be an empty string. Replacing _logical or_ with _nullish coalescing_ changes the behaviour of this part of the code for all the cases when fileName is an empty string. With '' as fileName and _logical or_ `FileUtils.getFileName(url)` is called which is not a case when `||` is replaced with `??`. Would you like me to include this explanation in the comment or do you think it is clear as it is?

Reply:
Great, I have changed that